stories/Radio.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Radio as Component } from "../confetti-ds/src";
+
+export default {
+  title: "Radio/Radio",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
```

Comment:
these opptions are wrong. ["string", "number", "bool"] are **types** of values, not values. please remove this control.

Reply:
fixed